rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -754,6 +762,12 @@ stream {
         ssl_stapling                            on;
         ssl_stapling_verify                     on;
         {{ end }}
+
+        {{ if and (not $all.DisableLua) $all.DynamicConfigurationEnabled }}
+        ssl_certificate_by_lua_block {
+            lua_certificate.call()
```

Comment:
this should be `certificate` instead of `lua_certificate`, see https://github.com/Shopify/ingress/pull/60/files#diff-980db9e4b88704f12338bd074839f94eR87